chainerrl/experiments/train_agent_batch.py

```diff
@@ -86,9 +88,6 @@ def train_agent_batch(agent, env, steps, outdir, log_interval=None,
             #   5. reset the env to start a new episode
```

Comment:
Should these comments be revised?

Reply:
I think these comments are still correct except that 3-5 are skipped when training is finished. I'll clarify this in the comments.